webapp/README.md

```diff
@@ -33,48 +33,74 @@ $ yarn build
+$ docker-compose exec webapp yarn lint
 ```
-The output should look similar to this:
 
-![Storybook output](../.gitbook/assets/storybook-output.png)
+```bash
+# run unit tests
+$ docker-compose exec webapp yarn test
+```
 
-Click on the link http://localhost:3002/ to open the browser to your interactive storybook.
+```bash
+# start storybook
+$ docker-compose exec webapp yarn storybook
+```
+
+You can then visit the Storybook playground on `http://localhost:3002`
 
 {% endtab %}
 
 {% tab title="Without Docker" %}
-Run the following command:
+
+After starting the application following the above guidelines open new terminal windows for each of these commands:
 
 ```bash
-# in webapp/
-yarn storybook
+# run eslint
+$ yarn lint
 ```
 
-Open http://localhost:3002/ in your browser
+```bash
+# run unit tests
+$ yarn test
+```
 
-{% endtab %}
-{% endtabs %}
+```bash
+# start storybook
+$ yarn storybook
+```
 
+You can then visit the Storybook playground on `http://localhost:3002`
 
+{% endtab %}
+{% endtabs %}
 
-## Styleguide
+## Styleguide Migration
 
-All reusable Components \(for example avatar\) should be done inside the [Nitro-Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) repository.
+We are currently in the process of migrating our styleguide components and design tokens from the [Nitro Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) into the main [Human Connection repository](https://github.com/Human-Connection/Human-Connection) and refactoring our components in the process. During this migration our new components will live in a `view` folder to separate them from the old, yet untouched components.
 
-![Styleguide Screenshot](../.gitbook/assets/screenshot-styleguide%20%281%29.png)
+### Folder Structure
 
-More information can be found here: [https://github.com/Human-Connection/Nitro-Styleguide](https://github.com/Human-Connection/Nitro-Styleguide)
+The folder structure we are aiming for is based on the [directory setup proposed by Nuxt.js](https://nuxtjs.org/guide/directory-structure):
 
-If you need to change something in the styleguide and want to see the effects on the frontend immediately, then we have you covered. You need to clone the styleguide to the parent directory `../Nitro-Styleguide` and run `yarn && yarn run dev`. After that you run `yarn run dev:styleguide` instead of `yarn run dev` and you will see your changes reflected inside the frontend!
+- **assets** contains icons, images and logos in `svg` format
+- **components** are the generic building blocks of the app – small, reusable and usually not coupled to state
+- **features** are composed of components but tied to a particular function of the app (e.g. `comment` or `post`)
+- **layouts** can use components to create layout templates for pages
+- **pages** are the entry points for all `routes` in the app and are composed of layouts, features and components
+- **styles** holds all shared SCSS files such as `variables` and `mixins`
```

Comment:
I haven't seen the `features` folder so far - @alina-beck have you been inspired by some other projects or did you come up with this on your own?

Reply:
I know the `features` folder from some React projects – although they often include more than just components ([here, for example](https://ryanlanciaux.com/blog/2017/08/20/a-feature-based-approach-to-react-development/)).

I guess it's just a matter of naming things. (_Eaaasy!_ :grin:) Some call it `containers` and `components`, others `components` and `atoms`, others `shared` and `modules`... I picked `features` and `components` because it seemed more descriptive to me. What are your preferences @mattwr18 and @roschaefer?

The idea is, as @mattwr18 already pointed out, to separate `stateful` from `stateless` components, or rather: to separate 'dumb' generic components from the ones knowing which data they are handling.